sdk/src/main/java/com/horizen/utils/MerklePathSerializer.java

```diff
@@ -5,8 +5,8 @@
 import scorex.util.serialization.Writer;
 import java.util.ArrayList;
 
-public class MerklePathSerializer<T extends MerklePath> implements ScorexSerializer<MerklePath> {
-    private static MerklePathSerializer<MerklePath> serializer;
+public class MerklePathSerializer<T extends MerklePath> implements ScorexSerializer<T> {
```

Comment:
Should be:
`public class MerklePathSerializer implements ScorexSerializer<MerklePath>`